### PR TITLE
fix prints and batchnorm

### PR DESCRIPTION
**Pull request by eyaler**
changed prints to functions where missing to work with python 3 and changed batchnorm function to work with tf 0.12 (see https://github.com/yenchenlin/pix2pix-tensorflow/issues/6).
i took the code from here: https://github.com/carpedm20/DCGAN-tensorflow/blob/master/utils.py
my previous pr was referencing the wrong source branch

### CODE REVIEW

**Comment by yenchenlin** [comment]
Thanks!

**Comment by HCShi** [comment]
Thanks!
